packages/lab/src/Flow/stories/BaseHook/index.tsx

```diff
@@ -0,0 +1,234 @@
+} from "@hitachivantara/uikit-react-lab";
+
+// The code for these values are available here: https://github.com/lumada-design/hv-uikit-react/tree/master/packages/lab/src/components/Flow/stories/Base/index.tsx
+import { restrictToSample } from "../Base";
+import { Asset } from "../Base/Asset";
+import { Dashboard } from "../Base/Dashboard";
+import { KPI } from "../Base/KPI";
+import { LayoutsProvider } from "../Base/LayoutsContext";
+import { LineChart } from "../Base/LineChart";
+import { MLModel } from "../Base/MLModel";
+import { Table } from "../Base/Table";
+import { AssetDA } from "./AssetDA";
+import { AssetWorkbench } from "./AssetWorkbench";
+import { DANode } from "./DANode";
+import { MLModelDA } from "./MLModelDA";
+import { WebttleNode } from "./WebttleNode";
+
+const nodeGroups = {
+  assets: {
+    label: "Asset",
+    color: "cat3_80",
+    description: "Find here all the available assets.",
+    icon: <Cluster />,
+    items: [
+      {
+        nodeType: "asset",
+        label: "My Asset",
+        data: { asset: "option1" },
+      },
+      {
+        nodeType: "assetDA",
+        label: "My DA",
+        data: { asset: "option1" },
+      },
+      {
+        nodeType: "webttle",
+        label: "My Webttle",
+      },
+      {
+        nodeType: "assetWorkbench",
+        label: "My Asset Workbench",
+        data: { asset: "option1" },
+      },
+    ],
+  },
+  models: {
+    label: "ML Model",
+    color: "cat1_80",
+    description: "Find here all the available ML models.",
+    icon: <MachineLearning />,
+    items: [
+      {
+        nodeType: "model",
+        label: "ML Model Prediction",
+        data: { type: "prediction" },
+      },
+      {
+        nodeType: "model",
+        label: "ML Model Detection",
+        data: { type: "detection" },
+      },
+      {
+        nodeType: "modelDA",
+        label: "ML Model DA",
+        data: { type: "detection" },
+      },
+    ],
+  },
+  insights: {
+    label: "Insight",
+    color: "cat6_80",
+    description: "Find here all the available insights.",
+    icon: <LineChartAlt />,
+    items: [
+      { nodeType: "kpi", label: "KPI" },
+      { nodeType: "lineChart", label: "Line Chart" },
+      { nodeType: "table", label: "Table" },
+    ],
+  },
+  da: {
+    label: "My Asset DA",
+    description: "Find here all the available assets.",
+    color: "cat6_80",
+    items: [{ nodeType: "DA", label: "My Asset" }],
+  },
+  webttle: {
+    label: "My Asset Webttle",
+    color: "cat3_80",
+    icon: <MachineLearning />,
+    items: [{ nodeType: "webttle", label: "My Asset" }],
+  },
+} satisfies HvFlowProps["nodeGroups"];
+
+export type NodeGroup = keyof typeof nodeGroups;
+
+export const nodeTypes: HvFlowProps["nodeTypes"] = {
+  asset: Asset,
+  model: MLModel,
+  kpi: KPI,
+  lineChart: LineChart,
+  table: Table,
+  dashboard: Dashboard,
+  DA: DANode,
+  webttle: WebttleNode,
+  assetDA: AssetDA,
+  modelDA: MLModelDA,
+  assetWorkbench: AssetWorkbench,
```

Comment:
shouldn't we focus only on the nodes built with the hook? I think adding all the other nodes adds entropy